master/docs/relnotes/index.rst

```diff
@@ -36,6 +36,10 @@ Fixes
 
 * OpenStackLatentWorker uses the novaclient API correctly now.
 
+* runGlob() uses the correct remote protocol for both :py:class:`~buildbot.process.buildstep.CommandMixin` and :py:class:`~buildbot.steps.worker.ComposititeStepMixin`.
```

Comment:
Use verbatim formatting for code blocks or code identifiers:

```
``runGlob()``
```

here and below.